packages/backend/src/App.ts

```diff
@@ -400,13 +417,14 @@ export default class App {
             analytics: this.analytics,
             // TODO: Do not use serviceRepository singleton:
```

Comment:
Left the comment here, since we continue to use what's functionally a singleton here, and ultimately we don't want to.